### PR TITLE
Systray fuctionality

### PR DESCRIPTION
**Pull request by luishutterli**
Added Util to run the page switcher in a systray application on windows.
Features:
```
Systray
Pause/Unpause of automatic switching in the systray menu
Most errors get handeld so the application doesn`t crash and can run continuously
Directly open the configurator site from the systray
```

Goals:
```
Make a programm that is enough stable so  it can be an auto-start programm that will run without crashing
```

Example Image of the systray application running
image

### CODE REVIEW

**Review by koriwi — CHANGES_REQUESTED** [review]
this seems to remove all linux and macos functionality. This regression is not acceptable

**Comment by luishutterli** [comment]
I have now changed it so there are still the old files for linux, mac and linux but that the new systray app is in its own folder

File: README.md [hunk 1]
```diff
@@ -13,7 +13,8 @@ You can pick between the python, powershell and bash version. The pythonscript i
 2. Check if it's installed correctly by typing `python3 --version` in a console. It should show something like `Python 3.9.1`
 3. Navigate a console to the folder containing the script and type `pip install -r requirements.txt --user`
 4. Configure the page_list.txt as described below
-5. Launch the `auto_page_switch.py` script. Either by double clicking on it or by navigating a console to the folder that contains the script. Then type `python auto_page_switch.py`
+5. Launch the `auto_page_switch.py` script or the `systray_app.py`(only works windows). Either by double clicking on it or by navigating a console to the folder that contains the script. Then type `python auto_page_switch.py` or `python systray_app.py`(only works on windows)
```
Comment (legomannetje) [inline README.md:16]: (Only works on windows) instead of (only works windows)

**Review by legomannetje — CHANGES_REQUESTED** [review]
Please add a pip-requirements.txt file with all the used dependencies.

**Comment by luishutterli** [comment]
its the same packages as the normal python script uses so it would be the same requirements file

**Review by koriwi — CHANGES_REQUESTED** [review]
please remove the whitespace in the filepath "Windows SysTray" -> "Windows_SysTray"

also check your line endings. Windows changes the line endings https://www.cs.toronto.edu/~krueger/csc209h/tut/line-endings.html#:~:text=Text%20files%20created%20on%20DOS,(%22%5Cn%22).

**Comment by luishutterli** [comment]
Ok fixed that now!

**Pull request merged**